tests/text_test.py

```diff
@@ -248,6 +248,38 @@ def test_sequences_to_texts():
                           'the dog sat on the log',
                           'dogs <unk> <unk> <unk> <unk>']
 
+def test_sequences_to_matrix():
+    # Tests for the 'count' mode NumPy data type
+    int_types = [
+                (8, np.uint8),
+                (16, np.uint16),
+                (32, np.uint32),
+                (64, np.uint64)
+                ]
+    for i in range(len(int_types) - 2):
```

Comment:
@Dref360 the reason why this for loop only goes up to -2 is due to the fact that generating strings with a number of words in the order of 2**32 or more takes too much memory.